pkgs/applications/misc/bikeshed/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, pytestCheckHook
+# build inputs
+, aiofiles
+, aiohttp
+, attrs
+, certifi
+, cssselect
+, html5lib
+, isodate
+, json-home-client
+, lxml
+, pillow
+, pygments
+, requests
+, result
+, setuptools
+, tenacity
+, widlparser
+}:
+
+buildPythonApplication rec {
+  pname = "bikeshed";
+  version = "3.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-eIr6AxBgX/cArO9qEYG5jnSiaDIddYiDbAYNqjcnrNc=";
+  };
+
+  # Relax requirements from "==" to ">="
+  # https://github.com/tabatkins/bikeshed/issues/2178
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "==" ">="
+  '';
+
+  propagatedBuildInputs = [
+    aiofiles
+    aiohttp
+    attrs
+    certifi
+    cssselect
+    html5lib
+    isodate
+    json-home-client
+    lxml
+    pillow
+    pygments
+    requests
+    result
+    setuptools
+    tenacity
+    widlparser
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  checkPhase = ''
+    $out/bin/bikeshed test
+  '';
```

Comment:
If we are not using pytestCheckHook then we shouldn't use it as an input. Does this even use pytest now?

Reply:
I tried `pytest` on either of those, but it didn't find any tests. So I removed it now from the nix files.